common/web/types/src/kpj/keyman-developer-project.ts

```diff
@@ -108,6 +108,11 @@ export class KeymanDeveloperProjectOptions {
   compilerWarningsAsErrors: boolean = false;
   warnDeprecatedCode: boolean = true;
   checkFilenameConventions: boolean = false;  // missing option defaults to False
+  /**
+   * Skip building .keyboard_info and .model_info files, for example in
+   * unit tests or for legacy keyboards
```

Comment:
So will unit tests need to specify skipMetadataFiles=true?

Reply:
The ones that need it will do it. Currently, it's not causing trouble.